### PR TITLE
feat(select): select/button

### LINKED ISSUE #22: [Feat]: Add select/deselect button and save button
### Is your feature request related to a problem? Please describe.

We need to have 3 more buttons on the filter modal that include select all, deselect all, and save button.

### Describe the solution you'd like.

I would like fully functional buttons for the select all, deselect all, and save.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by itschrisboiii**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Changes
_List Changes Introduced by this PR_
1. added select and deselect buttons

## Purpose
_Describe the problem or feature in addition to a link to the issues._
i added select and deselect buttons for the filtered data 
## Approach
_How does this change address the problem?_
it makes sure a user can easily select and deselect interns

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #22

### CODE REVIEW

File: style/styles.css [hunk 1]
```diff
@@ -171,6 +171,59 @@ tbody td {
...
+    content: '✖'; /* Unicode character for checkmark */
+    color: black; /* Checkmark color */
+    font-size: 20px; /* Size of the checkmark */
+    position: absolute; /* Position the checkmark */
+    top: 0; /* Center vertically */
+    left: 2px; /* Center horizontally */
+}
+
+.buttonSelectStyle {
+    width: 144px;
```
Comment (ItzTeeler) [inline style/styles.css:201]: Those are a lot of comments.

Comment (ItzTeeler) [reply style/styles.css:201]: Unicode is cool!

Comment (itschrisboiii) [reply style/styles.css:201]: dispatchEvent() method allows you to trigger events programmatically on a DOM element. This means you can simulate user actions like clicks, key presses, or any custom events you define.

**Review by joavelar — APPROVED** [review]
Keep the good work!

**Pull request merged**